.github/workflows/makemhr.yml

```diff
@@ -0,0 +1,76 @@
+name: makemhr
+
+on:
+  push:
+    paths:
+      - 'utils/makemhr/**'
+      - '.github/workflows/makemhr.yml'
+
+  workflow_dispatch:
+
+env:
+  BUILD_TYPE: Release
+
+jobs:
+  Win64:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Get current date
+      run: echo "CurrentDate=$(date +'%Y-%m-%d')" >> $env:GITHUB_ENV
+
+    - name: Get commit hash
+      run: echo "CommitHash=$(git rev-parse --short=7 HEAD)" >> $env:GITHUB_ENV
+
+    - name: Clone libmysofa
+      run: git clone https://github.com/hoene/libmysofa.git libmysofa
```

Comment:
It may be a good idea to add `--depth 1` here since it doesn't need to get the entire history. It may also be a good idea to grab a specific release commit so that it's not prone to build failures or run-time bugs that may creep into the latest commit.

Reply:
Makes sense
Workflow updated to grab only the latest release tag https://github.com/hoene/libmysofa/releases/tag/v1.3.1